### PR TITLE
Brig Cell timers QoL changes.

### PR DESCRIPTION
**Pull request by Birdtalon**
I was asked to work on something along these lines by Spartan a while back but never really got around to it.

- Adds `seconds_to_time()` proc, which accepts a value of seconds as an argument and returns a string of `X minute(s) and X seconds`
- Replaces `-- ++` incremental controls with a single button which prompts two input boxes, one for minutes and seconds.
- Replaces values in seconds with minutes and seconds on the radio output and the printouts from security record computers.

Please note in the screenshot below I have replaced **"Set Timer"** with **"Input Time"** for a bit more clarity.

dreamseeker_2017-09-27_23-57-24
dreamseeker_2017-09-27_23-57-35

🆑 Birdtalon
tweak: User interface tweaks to Brig timers.
tweak: Seconds converted to minutes and seconds in Brig radio and security console printouts.
/🆑

### CODE REVIEW

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -55,7 +55,7 @@
 		cell_logs += P
 
 	var/datum/data/record/R = find_security_record("name", logname)
-	Radio.autosay("Detainee [logname] has been incarcerated for [timetoset/10] seconds for the charges of, '[logcharges]'. \
+	Radio.autosay("Detainee [logname] has been incarcerated for [seconds_to_time(timetoset / 10)] seconds for the charges of, '[logcharges]'. \
```
Comment (Birdtalon) [inline code/game/machinery/doors/brigdoors.dm]: I need to address the repeated word `seconds` here. Will do so tomorrow

**Comment by Spartan6** [comment]
I love you so much.

**Comment by Birdtalon** [comment]
Also forgot to add a changelog but I can't seem to edit on my phone so I'll update that tomorrow also.

**Comment by Anasari** [comment]
omae wa mou brigdeiru.

An amazing QOL change that reduce the amount of time needed to brig a person. I approve.

**Comment by tigercat2000** [comment]
LGTM

**Pull request merged**